yaml/roads.yaml

```diff
@@ -251,7 +251,7 @@ filters:
       kind_detail: {col: highway}
   - filter:
       highway: secondary
-    min_zoom: { clamp: { min: 0, max: 10, value: { call: { func: util.calculate_path_major_route, args: [ { col: fid } ] } } } }
```

Comment:
For this and all other changes in this file...

To match up with Natural Earth changes I'd expect this to be a clamp with min of 5 which is when we want to introduce any roads layer content.

Then we need to pair it with a zoom transition in the OpenStreetMap JINJA logic.

```
    min_zoom: { clamp: { min: 5, max: 10, value: { call: { func: util.calculate_path_major_route, args: [ { col: fid } ] } } } }
```